config/autoconfigbackup/autoconfigbackup.inc

```diff
@@ -124,29 +151,30 @@ function test_connection($post) {
 	curl_setopt($curl_session, CURLOPT_POSTFIELDS, "action=showbackups&hostname={$hostname}");
 	$data = curl_exec($curl_session);
 
-	if (curl_errno($curl_session)) 
+	if (curl_errno($curl_session)) {
 		return("An error occurred " . curl_error($curl_session));
-	else 
+	} else {
 		curl_close($curl_session);
-
+	}
 	return;
 }
 
 function upload_config($reasonm = "") {
 	global $config, $g, $input_errors;
 
-	/* do nothing when booting */
-	if($g['booting']) 
+	// Do nothing when booting
+	if (platform_booting()) {
```

Comment:
I needs to be checked as it was done in https://github.com/pfsense/pfsense-packages/pull/1021/files#diff-a19fa0627590e2e997660ff75d27a301R112